src/index.ts

```diff
@@ -273,10 +273,9 @@ export function serializeBufferFromStorage(str: string): Uint8Array {
  */
 export function serializeBufferForStorage(buffer: Uint8Array): string {
   let result = '0x';
-  const len = buffer.length || buffer.byteLength;
-  for (let i = 0; i < len; i++) {
-    result += unprefixedHex(buffer[i]);
-  }
+  buffer.forEach((value) => {
+    result += unprefixedHex(value);
+  });
   return result;
 }
```

Comment:
This entire function could be replaced with `bytesToHex` from `@metamask/utils`.

Reply:
Nice! I'm tempted to leave this inline for now until we migrate this into the monorepo, to avoid the dependency bloat problem of having incompatible util library versions (the monorepo basically solves this by making it possible for them to be kept in sync). I'll create an issue for this though, thanks for pointing it out.

Reply:
https://github.com/MetaMask/browser-passworder/issues/30

Reply:
We could make it a peer dependency if you don't want dependency bloat? The version in the extension has the bytes functions already, a lot of functions in this library could be replaced.

Reply:
The dependency bloat problem I had in mind was where we need multiple different copies due to major version updates of the utils package, which leaves us with multiple copies unless we're very diligent with updating all packages that use utils (sometimes with can take up to 4 levels of coordinated releases). Adding it as a peer dependency just moves that problem up one level. With the monorepo we are planning on using a Yarn constraint to ensure we have compatible versions across the board.

I've been holding off on replacing internal functions with utils for this reason. The monorepo should land tomorrow or Monday though, and I am hoping that migrating a package into it will be a straightforward process. So we may be able to migrate this into the monorepo as early as next week.